src/domain/bitcoin/lightning/index.types.d.ts

```diff
@@ -67,30 +86,57 @@ type LnFeeCalculator = {
   max(amount: Satoshis): Satoshis
 }
 
+type PayInvoiceResult = {
+  roundedUpFee: Satoshis
+}
+
 interface ILightningService {
   isLocal(pubkey: Pubkey): boolean | LightningServiceError
+
+  defaultPubkey(): Pubkey
+
+  lndFromPubkey(pubkey: Pubkey): AuthenticatedLnd | LightningServiceError
```

Comment:
Tho whole client-side load-balancing / pub key stuff needs to be handled internally to the service! We should not expose any of it to the outside layers.